game/fighter.py

```diff
@@ -34,6 +35,14 @@ def __init__(self, player, x, y, flip, punch_sound, projectile_sound, hit_sound,
         self._start_x = x
         self._start_y = y
         self.jump_height = 20
+        self.player_x = x
+        self.player_y = y
+        self.ai = ai
+        self.moves = [0,0,0,0,0,0]
+
+    def return_state(self):
+        return  [self.player_x,self.player_y,self.vel_x,self.vel_y, self.shooting_projectile, self.attacking,
+                        self.running , self.jump , self.blocking]
```

Comment:
this ai stuff should not be in this fighter class, your fighter class should inherit from this fighter class and add on extra features

Reply:
the return state is needed for the ai to know the fighters position and current action